packages/integrations/deno/CHANGELOG.md

```diff
@@ -32,7 +40,7 @@
   The new `Astro.clientAddress` property allows you to get the IP address of the requested user.
 
   ```astro
-  <div>Your address {Astro.clientAddress}</div>
```

Comment:
Was this line removed on purpose?

Reply:
Nope, see convo happening here https://github.com/withastro/astro/pull/4214#discussion_r941460217